packages/api/src/controllers/session.ts

```diff
@@ -185,6 +185,6 @@ function removePrivateFields(obj: DBSession) {
 
 const adminOnlyFields: (keyof DBSession)[] = ["deleted", "broadcasterHost"];
 
-const privateFields: (keyof DBSession)[] = ["recordObjectStoreId", "version"];
```

Comment:
we should probably still keep this since the legacy objects are still in the DB and we can't return this field (apparently)

Reply:
Reverted in 8b722695100a095805efb8ec6f9d3cb3bee1ec23